Umbraco Cloud - Self service WAF features (#7322)
umbraco/UmbracoDocs: umbraco-cloud/build-and-customize-your-solution/set-up-your-project/security/managing-transport-security.md

A Web Application Firewall (WAF) is a security solution designed to protect web applications by filtering and monitoring HTTP traffic between them and the Internet. Common attacks like cross-site scripting, SQL injection, and file inclusion are mitigated by acting as a shield between the web application and potential threats. For more detailed information, please refer to our [WAF section](web-application-firewall.md).

## Web Application Firewall Sensitivity

The Umbraco Cloud WAF supports sensitivity configuration, extending the default [WAF](web-application-firewall.md) protection. The default [WAF](web-application-firewall.md) and WAF sensitivity configuration options don't interact and can be controlled separately. Customers are encouraged to experiment with the WAF configuration at the beginning of the project and adjust based on the performance.

- **Low** severity configuration will block malicious requests with high confidence - blocks less requests.
- **High** severity configuration will block malicious requests with medium confidence, providing stricter filtering - blocks more requests. 
- **Off** configuration will not block any requests.

## Managed Challenge

> A managed challenge is a lightweight JavaScript-based page that detects users without user inputs. After successfully passing a challenge, the user will receive a cookie. Users with a cookie won't be asked to pass another challenge for 30 minutes anywhere on the project/hostname.

Reviewer: ```suggestion
A managed challenge is a lightweight JavaScript-based page that detects users without user input. After successfully passing a challenge, the user will receive a cookie. Users with a cookie won't be asked to pass another challenge for 30 minutes anywhere on the project/hostname.
```

> Enabling the Managed Challenge presents an automatic captcha to all requests for the project/hostname. The managed challenge will ensure that only human users will be able to access the content on the website. Presenting an automatic managed captcha is useful in cases when a website is experiencing higher load. Higher load on the website can be caused by any reason such as, DDoS attack, aggressively scraped by bots or in high demand. Enabling a managed challenge will ensure that all of your website's resources are delivering value to the end users.

Reviewer: ```suggestion
Enabling the Managed Challenge presents an automatic CAPTCHA to all requests for the project/hostname. The managed challenge will ensure that only human users will be able to access the content on the website. Presenting an automatically managed CAPTCHA is useful in cases when a website is experiencing higher load. Higher load on the website can be caused by any reason, such as, DDoS attack, aggressive scraped by bots, or high demand. Enabling a managed challenge will ensure that all of your website's resources are delivering value to the end users.
```

## Continent Managed Challenge

Selecting continent(s) in the list will present a managed challenge to all traffic from the continent(s).

> Continent-based managed challenge presents a challenge meant to only pass through the human users requesting the website from selected continents. A continent-based managed challenge is useful when a website's primary users live in a specific continent. By presenting a challenge to selected continents you can block all malicious traffic from the continent while allowing humans to pass through.

Reviewer: ```suggestion
Continent-based managed challenge presents a challenge meant to only pass through the human users requesting the website from selected continents. A continent-based managed challenge is useful when a website's primary users live on a specific continent. By presenting a challenge to selected continents, you can block all malicious traffic from the continent while allowing humans to pass through.
```

> All continents are supported as well as presenting a challenge to all requests from the [Tor](https://en.wikipedia.org/wiki/Tor_(network)) network.

Reviewer: ```suggestion
All continents are supported, as well as presenting a challenge to all requests from the [Tor](https://en.wikipedia.org/wiki/Tor_(network)) network.
```
